exploration/quoted-literals.md

```diff
@@ -155,6 +156,16 @@ _What properties does the solution have to manifest to enable the use-cases abov
   However, quoted literals are usually short and already enclosed in a placeholder (which has its own delimiters)
   or are outside patterns (when used as variant keys).
 
+- **[r6; low priority]** Be able to drop messages into host formats without changing them.
+  For example, if the host format uses `""` to delimit strings literals,
+  then all occurences of `"` inside messages must be escaped.
+  While we can't prevent `"` from appearing in translation content,
+  we can minimize the overhead of the MessageFormat 2 syntax.
+  This requirement is scored as _low_, because many storage formats don't use delimiters at all (`.properties`, YAML),
+  or they are meant to be primarily used by machines (JSON),
+  and because many programming languages provide a way to delimit _raw strings_,
+  e.g. via backticks in JavaScript and `"""` in Python.
+
```

Comment:
```suggestion
  Note that messages including e.g. newlines or `\` escapes in their source
  will likely need those characters accounted for when dropping them into new host formats.

```

Reply:
Eemeli and I met, and we included this suggestion in the PR.